tests/test_utils.py

```diff
@@ -89,10 +86,49 @@ def make_links_blue(node):
                 "children": ["universe"],
                 "attributes": {"style": {"color": "blue"}},
             },
+            "lmao",
+        ],
+    }
+
+    assert html_to_vdom(source, make_links_blue) == expected
+
```

Comment:
This should get the last bit of coverage.

```suggestion


def test_non_html_tag_behavior():
    source = "<my-tag data-x=something><my-other-tag key=a-key /></my-tag>"

    expected = {
        "tagName": "my-tag",
        "attributes": {"data-x": "something"},
        "children": [
            {"tagName": "my-other-tag", "key": "a-key"},
        ],
    }

    assert html_to_vdom(source) == expected

```